docs/docs/reference/contextual/derivation.md

```diff
@@ -3,360 +3,355 @@ layout: doc-page
 title: Typeclass Derivation
 ---
 
-Typeclass derivation is a way to generate given instances for certain type classes automatically or with minimal code hints. A type class in this sense is any trait or class with a type parameter that describes the type being operated on. Commonly used examples are `Eql`, `Ordering`, `Show`, or `Pickling`. Example:
+Type class derivation is a way to automatically generate given instances for type classes which satisfy some simple
```

Comment:
Is it "typeclass" or "type class"? The spelling is inconsistent in this file, starting from the title.

Reply:
It's "type class". I'll check for misspellings and correct.

Reply:
I don't want to start a debate but I'm curious, why "type class" and not "typeclass" ? :) I like the latter because it makes it slightly less likely that beginners would confuse them with the notion of class in Scala.

Reply:
I don't want to start a debate but I'm curious, why "type class" and not "typeclass" ? :) I like the latter because it makes it slightly less likely that beginners would confuse them with the notion of class in Scala.

Reply:
The canonical references all use "type class": https://homepages.inf.ed.ac.uk/wadler/topics/type-classes.html. It's conventional, rather than something which is either right or wrong. I prefer to go with Wadler, but reasonable people might disagree.

Reply:
Whichever you choose the title should be updated to match.

Reply:
Opinion from the peanut gallery: I tend to see "typeclass" more often nowadays (and use it that way myself).

IMO, the single-word form is gradually becoming fairly standard jargon.  But as you say, there's plenty of room for reasonable disagreement.